static/css/perdiem.css

```diff
@@ -546,16 +546,43 @@ span.text-content {
     background: rgba(0,0,0,0.5);
 }
 
-span.text-content span {
+span.text-content-hide-text span {
     display: table-cell;
     text-align: center;
     vertical-align: middle;
 }
 
-.img-list:hover span.text-content {
+.img-list:hover span.text-content-hide-text {
     opacity: 0;
 }
 
+span.text-content-reveal-text {
```

Comment:
Rather than copying `text-content-hide-text` (I renamed it from `text-content`) and changing only the opacity, maybe we can create a common CSS class that would be used in both use cases. And then `text-content-reveal-text` can just have the `opacity: 1;` value whereas `text-content-reveal-text` has just the `opacity: 0;` value.